conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -15,11 +15,8 @@ POST       /test-only/sbt                   @uk.gov.hmrc.servicedependencies.tes
 POST       /test-only/library               @uk.gov.hmrc.servicedependencies.testonly.IntegrationTestController.addLibraryVersion
 POST       /test-only/dependencies          @uk.gov.hmrc.servicedependencies.testonly.IntegrationTestController.addDependencies
 POST       /test-only/sluginfos             @uk.gov.hmrc.servicedependencies.testonly.IntegrationTestController.addSluginfos
-POST       /test-only/slugDependencyConfigs @uk.gov.hmrc.servicedependencies.testonly.IntegrationTestController.addSlugDependencyConfigs
-DELETE     /test-only/sbt                   @uk.gov.hmrc.servicedependencies.testonly.IntegrationTestController.deleteSbt
```

Comment:
What was the endpoint sbt for?

Reply:
Accidentally removed, I've restored it now.  (It's not actually currently used in the acceptance tests, but I guess it might be useful in manual testing.)